app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
```suggestion
    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
```
chỗ này nên dùng blank? sẽ ok hơn bé nha